config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml

```diff
@@ -340,7 +340,6 @@ spec:
             - currentStatefulSetReplicas
             - mongoUri
             - phase
-            - version
```

Comment:
looks like this field was manually removed from the required list by this [PR 684](https://github.com/mongodb/mongodb-kubernetes-operator/pull/684/files) but we missed to update the type (tag the version field with "omitempty")